src/pybind/mgr/orchestrator/module.py

```diff
@@ -2250,6 +2250,21 @@ def _tuned_profile_rm_setting(self, profile_name: str, setting: str) -> HandleCo
         res = raise_if_exception(completion)
         return HandleCommandResult(stdout=res)
 
+    @_cli_write_command('orch tuned-profile add-settings')
+    def _tuned_profile_add_settings(self, profile_name: str, settings: str) -> HandleCommandResult:
+        setting_pairs = settings.split(',')
+        parsed_settings = {key.strip(): value.strip() for key, value in (s.split('=', 1) for s in setting_pairs)}
```

Comment:
we should probably have this in a try-except block and print a usage message explaining the format for calling this command if we find an issue with what the user passed in